conan_support.cmake

```diff
@@ -13,6 +13,14 @@ function(detect_os OS)
 endfunction()
 
 
+function(detect_os_version OS_VERSION)
```

Comment:
If this is coupled to the detection of the OS, I would make it part of ``detect_os``, and make that return both MYOS and MYOS_VERSION

Reply:
Done